plugins/core/jetbrains-community/resources/META-INF/aws.toolkit.core.xml

```diff
@@ -68,7 +68,7 @@
         <postStartupActivity implementation="software.aws.toolkits.jetbrains.core.AwsTelemetryPrompter"/>
 
         <registryKey key="aws.dev.useDAG" description="True if DAG should be used instead of authorization_grant with PKCE"
-                     defaultValue="false" restartRequired="false"/>
+                     defaultValue="true" restartRequired="false"/>
```

Comment:
i’m not sure if this will work if the user previously has started the IDE when this was “false”

Reply:
changed registry name in the new commit